docs/collectors/PingCollector.md

```diff
@@ -52,5 +53,7 @@ use_sudo | False | Use sudo? | bool
 
 ```
 servers.hostname.ping.localhost 11
+servers.hostname.ping.www_example_org 23
```

Comment:
This section of the doc file is generated from the `test_should_work_with_real_data_host_gentoo` method in the test file so this section is actually correct.  If you want to add more examples, please update the test method.